common/src/test/java/com/datastrato/gravitino/dto/requests/TestTableUpdatesRequest.java

```diff
@@ -102,6 +102,11 @@ public void testTableUpdatesRequest() throws JsonProcessingException {
             + "}";
     Assertions.assertEquals(
         JsonUtils.objectMapper().readTree(expected), JsonUtils.objectMapper().readTree(jsonString));
+
+    // test validate blank property value
+    TableUpdateRequest.SetTablePropertyRequest setTablePropertyRequest =
+        new TableUpdateRequest.SetTablePropertyRequest("key", " ");
+    Assertions.assertDoesNotThrow(setTablePropertyRequest::validate);
```

Comment:
We also need to check “”

Reply:
what do you mean? we should allow `""` or not?

Reply:
Add test case of "".

Reply:
added